meta/main.yml

```diff
@@ -10,7 +10,7 @@ galaxy_info:
   platforms:
   - name: Ubuntu
     versions:
-      - xenial
```

Comment:
does it matter to us that we can't rerun this for the other older apps? I don't think we want to so I'm not sure it matters